content/guides/figma/getting-started.mdx

```diff
@@ -18,30 +19,81 @@ You can also directly open the *team library* view via the command palette or wi
   />
 </ImageBox>
 
+### Styles and variables
 
-## Understanding styles
-Figma libraries like `Primer Primitives` install styles for you to use. In contrast to *local styles* they don't show up in the sidebar.
-However once you open a selection to choose a color style, text style, etc. you will see the styles from team libraries as well.
+Styles and variables are two very similar ways of bringing `design tokens` like [Primer Primitives](https://github.com/primer/primitives) into Figma.
+Our goal is to move everything over to variables, but at the moment only color and size tokens are supported. This is why we still provide text and shadow tokens using styles.
 
-To quickly find a style you can use the search box. E.g. use `accent` to bring up all accent colors.
+### Understanding styles
+Figma libraries like `Primer Web` provide styles for you to use. In contrast to local styles, styles from a library don't show up in the sidebar. However once you open a selection to choose a `text style` or `shadow` you will see the styles from team libraries as well.
 
-<ImageBox caption="Color style selector and color style selector with search for 'accent'">
+Available styles:
+- text styles
+- shadow styles
+
+To quickly find a style you can use the search box. E.g. search for `body` to bring up all body text styles as well as related ones.
+
+<ImageBox caption="Text style selector with search for 'body'">
   <img
-    width="443"
+    width="261"
     alt="Screenshot showing the style selector in figma"
-    src="https://user-images.githubusercontent.com/813754/221531315-605cba02-de29-4834-a01b-ef8dc391fe21.png"
+    src="https://github.com/primer/react/assets/813754/4537b229-a663-499d-b0f0-d6be0d9fdadb"
+  />
+</ImageBox>
+
+### Understanding variables (primitives)
+Our Figma libraries use `variables` to represent design tokens. Variables from a library don't show up in the sidebar.
+However once you open a selection to choose a color, or size, etc. you will see the variables from team library as well.
+
+To use the `variables` you need to enable the `Primer Web` library in your file.
+
+Available variables:
+- color variables
+- size variables
+
+To quickly find a `variable` you can use the search box. E.g. use `accent` to bring up all accent colors or `fgColor` to find all text related colors. 
+
+<ImageBox caption="Color variables panel and panel with a search for 'fgCo'">
+  <img
+    width="509"
+    alt="Screenshot showing the color variable panel in figma"
+    src="https://github.com/primer/design/assets/813754/34835137-cc62-45ce-9abf-3573089eb278"
   />
 </ImageBox>
 
-<a href="https://help.figma.com/hc/en-us/articles/360039238753-Styles-in-Figma">Learn more about Figma styles <LinkExternalIcon /></a>
+#### Scope
+Variables in Figma are *"scoped"*, meaning they can only be used in specific situations. Currently Figma offers the following scopes for color variables: `Frame fill`, `Shape fill`, `Text fill` , `Stroke`.
```

Comment:
Non-blocking nitpick: I'm not sure we need "Frame fill", "Shape fill", etc in `<code>` blocks. Those just refer to Figma tools.

Reply:
Same as above. I do feel it is nice to somehow highlight that those are actual scope values. But maybe a code block is not correct, and we should use a different visual style?

Out of interest, why do you feel code blocks are problematic here?

Reply:
I think it's just a matter of personal preference. It makes me thinks these are things we have in Primer code. If nobody else gets hung up on these, I think it's totally fine.